### ISSUE TITLE
Librams' acquisition and enchanting

### ISSUE THREAD

**Issue by Slamminn** [SunstriderEmu/BugTracker#3770]
Hello, I believe that there are a few bugs in regard to the Libram book enchants from both Dire-Maul and BRD (librams eg verocity, focus, etc etc).

**Describe the bug**
There are 2 bugs relating to these items.

1. When using the Cenarion plant salve's on the corrupted plants in Felwood, you can use the salves on the plants and get consumed on use, however the plants do not become un-corrupted (ie not spawning the in un-corrupted plant objects) and you can not loot them (so you can't complete any of the BRD librams)

2. When you have acquired a libram (using the dire-maul ones in this case (I tried the Libram of Focus)), you can not use it to enchant another persons gear (ie through the trade window), which is meant to be possible in the base game (sources below): When attempting to do so, you are given the error message "That item is not a valid target".
Sources:
http://www.realpapa.ru/hobby/wow/libram/librams.html
https://www.reddit.com/r/FairbanksClassic/comments/fdhand/a_selling_arcanum_enchants_applied_to_gear/

**To Reproduce**
(see above)

I believe all of these items will be bugged, but for testing, I have only attempted a few. 

For Bug 1:
Quest: https://classic.wowhead.com/object=164888/corrupted-whipper-root
(Doesn't spawn the un-corrupted version)
Object meant to be spawned:
https://classic.wowhead.com/object=164883/cleansed-whipper-root

For Bug 2:
Item: 
https://classic.wowhead.com/item=18330/arcanum-of-focus#comments

**Expected behavior**
(described above)

**Additional context**
Basically, most of these enchants are not obtainable (due to the current bugs) and the ones that you can acquire do not work as intended.

Cheers

**Comment by beraus** [comment]
1. https://github.com/SunstriderEmu/BugTracker/issues/3285

**Comment by Sheikthecatman** [comment]
Thanks! All Corrupted Flowers have been fixed in [3a6dac4a1b43db0290e815de6fa27463a2cad9e8](https://github.com/kelno/sunstrider-scripts/commit/3a6dac4a1b43db0290e815de6fa27463a2cad9e8).

The behavior should be corrected, the fix will be live next server restart.
